### DISCUSSION TITLE
FrSky Archer Sr8 on Tx16s

### DISCUSSION THREAD

**Discussion by RCAviation** [EdgeTX/edgetx#3233] [Q&A]
Hi,

I have a FrSky Archer SR8pro and a Radiomaster TX16s.
How do I bind setup this combination?
Do I have to flash the Archer with accst protocol? Set the Radiomaster to Rx9 protocol and bind? I really want to learn something here. 

Thanks in advance 

Marc 🇳🇱

**Comment by raphaelcoeffic** [comment]
I believe it will only work with FrSky's ISRM modules (those present in X10 Express or sold as upgrade internal modules), as these receivers don't have any ACCST firmware.

**Reply by RCAviation** [reply]
Is there a external module for sale I can mount in the back of my Tx16s?

**Reply by raphaelcoeffic** [reply]
Nope, none that would do ACCESS 2.4GHz, at least that I know of.
